protos/robots/nubots/NUgus/NUgus_mesh/NUgusMain.proto

```diff
@@ -0,0 +1,1716 @@
+                                        anchor 0.000000 0.040000 0.000000
+                                        dampingConstant IS MX106-damping
+                                        staticFriction IS MX106-friction
+                                      }
+                                      device [
+                                        RotationalMotor {
+                                          name "right_ankle_roll"
+                                          maxVelocity IS MX106-vel
+                                          minPosition -3.14159265359
+                                          maxPosition 3.14159265359
+                                          maxTorque IS MX106-torque
+                                        }
+                                        PositionSensor {
+                                          name "right_ankle_roll_sensor"
+                                          resolution IS DYNAMIXEL-RESOLUTION
+                                        }
+                                      ]
+                                      endPoint Solid {
+                                        translation 0.000000 0.040000 0.000000
+                                        rotation -0.000001 1.000000 -0.000001 3.141593
+                                        children [
+                                          DEF r_sole Solid {
+                                            rotation 0 0 1 -1.5708
+                                            translation -0.00825 -0.0645 -0.06
+                                          }
+                                          DEF right_foot NUgus_right_footMesh {
+                                            robotTexture IS robotTexture
+                                          }
+                                          # Define four touch sensors for the right foot
+                                          # Back right touch sensor on right foot
+                                          TouchSensor {
+                                            # Define translation of touch sensor
+                                            translation -0.0575 0.025 -0.061
+                                            children [
+                                              # Define shape for all touch sensors
+                                              DEF touch_shape Shape{
+                                                appearance PBRAppearance {
+                                                  baseColor 0.500000 0.500000 0.500000
+                                                  transparency 0.000000
+                                                  roughness 1.000000
+                                                  metalness 0
+                                                  emissiveColor 0.000000 0.000000 0.000000
+                                                  baseColorMap ImageTexture {
+                                                    url IS robotTexture
```

Comment:
I get an error I do not quite understand:
`WARNING: NUgus (PROTO) > HingeJointWithBacklash (PROTO) > Solid > HingeJointWithBacklash (PROTO) > Solid > HingeJointWithBacklash (PROTO) > Solid > HingeJointWithBacklash (PROTO) > Solid > HingeJointWithBacklash (PROTO) > Solid > HingeJointWithBacklash (PROTO) > Solid > HingeJoint > Solid > TouchSensor > DEF touch_shape Shape > PBRAppearance > ImageTexture: Cannot download 'https://raw.githubusercontent.com/cyberbotics/webots/R2022b/projects/joints/NUgus_textures/NUgus_base_color.jpg', error code: 203: Error transferring https://raw.githubusercontent.com/cyberbotics/webots/R2022b/projects/joints/NUgus_textures/NUgus_base_color.jpg - server replied: `

apparently, webots tries to fetch the texture from a server even though it is defined locally. Removing the baseColorMap did not seem to have an impact. What do you think @ysims